src/shared/helpers.js

```diff
@@ -0,0 +1,129 @@
+export const regExpEmail = /^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/i;
+export const regExpPassword = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/i;
+const regExpPhone = /^([+]?[\s0-9]+)?(\d{3}|[(]?[0-9]+[)])?([-]?[\s]?[0-9])+$/i;
+const regExpNumbers = /^[0-9,.]+$/i;
+const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
+
+export function createFullName(items) {
```

Comment:
this method should get as an input only 1 value, but not an array, rewrite it please

Reply:
ok, I understand. but this method removed.